pages/posts/bowtie-intro.md

```diff
@@ -12,24 +12,44 @@ authors:
     photo: /img/avatars/julian.webp
     link: https://twitter.com/JulianWasTaken
     byline: JSON Schema Test Suite Maintainer & Technical Lead, API Specifications @Postman
-excerpt: "Bowtie: a new tool for executing JSON Schema implementations & a call to help improve it"
+excerpt: "A new tool for executing JSON Schema implementations & a call to help improve it"
 ---
 
-*This is the first in what will hopefully be an intermittent series of posts about Bowtie.
-I speak here without speaking on behalf of the JSON Schema organization or its members in any official capacity -- Bowtie isn't a tool we The JSON Schema Team are "blessing" in some way today, though I have personal hopes it does become a sort of official tool, and have developed it fully intending for it to be owned by "the community" in whatever sense that means.
+*This is the first in what will hopefully be an intermittent series on Bowtie.
+I speak here without speaking on behalf of the JSON Schema organization in any official capacity -- Bowtie isn't a tool we The JSON Schema Team are "blessing" in some way today, though I have personal hopes it becomes a sort of official tool, and have developed it intending it to be owned by "the community" in whatever sense that means.
 For the moment, I speak only on behalf of myself as its author :)*
 
+The JSON Schema
+
+```json
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "string",
+  "maxLength": 3
+}
+```
+
+validates strings of length at most 3.
+This behavior is what the specification says *should* happen.
+Does every *implementation* of the specification, across the wide spectrum of programming languages, properly *follow* the specification?
+What about for other more complicated schema / instance pairs?
+This is the key question Bowtie tries to address -- how can we compare behavioral differences between JSON Schema implementations and between the specification, in the hopes that we make it easier to fix implementation issues and clarify any unclear portions of the specification?
+
 ## What It Is
 
 [Bowtie](https://github.com/bowtie-json-schema/bowtie/) is what I'm calling a "meta-validator", by which I mean a program which can execute *other* JSON Schema validation implementations and collect results from them.
 
+There's prior art in doing this sort of thing.
+The [JSON Path Comparison project](https://cburgmer.github.io/json-path-comparison/) does this extremely well for [JSONPath](https://goessner.net/articles/JsonPath/), and [Nicolas Seriot's "Parsing JSON is a Minefield"](https://seriot.ch/projects/parsing_json.html) is a fantastic example for JSON itself.
```

Comment:
I'm not hung up on it, but the official spec is being developed at https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base.  Alternatively this link can be used.  Although, the Goessner post is more well-known.  Just an idea.  I leave it to your preference.

Reply:
I have no preference, I took the link from the comparison page (which links to that Goessner one) -- happy to update it, I don't know anything about JSONPath's development but certainly trust you :D

Reply:
(Done)